tariff-management-service/app/build.gradle

```diff
@@ -30,6 +31,7 @@ dependencies {
     compile 'org.flywaydb:flyway-core:5.2.4'
     compile 'io.springfox:springfox-swagger2:2.8.0'
     compile 'io.springfox:springfox-swagger-ui:2.8.0'
+    compile 'com.microservices.tariffmanagement.api:api:0.1-SNAPSHOT'
```

Comment:
please move version to versions variables and please make it for all the services

Reply:
fixed. I had to use " instead of regular one to allow variables inserting